events/equeue/equeue_mbed.cpp

```diff
@@ -23,27 +23,36 @@
 #include <stdbool.h>
 #include "mbed.h"
 
+#if MBED_CONF_EVENTS_USE_LOWPOWER_TIMER_TICKER
+    #define AliasTimer      LowPowerTimer
```

Comment:
macros should start at the line beginning and should use capital letters , see https://docs.mbed.com/docs/mbed-os-handbook/en/latest/cont/code_style/

```
#if MBED_xxx
#define ALIAS_TIMER
```